assets/react/v3/entries/course-builder/components/additional/MeetingCard.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { format } from 'date-fns';
+
+import Button from '@Atoms/Button';
+import SVGIcon from '@Atoms/SVGIcon';
+
+import { borderRadius, colorTokens, fontWeight, spacing } from '@Config/styles';
+import { typography } from '@Config/typography';
+import Show from '@Controls/Show';
+
+import { styleUtils } from '@Utils/style-utils';
+import { __ } from '@wordpress/i18n';
+
+interface MeetingCardProps {
+  meeting_title: string;
+  meeting_date: string;
+  meeting_start_time: string;
+  meeting_token?: string;
+  meeting_password?: string;
+  meeting_link: string;
+}
+
+const MeetingCard = ({
+  meeting_title,
+  meeting_date,
+  meeting_start_time,
+  meeting_token,
+  meeting_password,
+  meeting_link,
+}: MeetingCardProps) => {
+  const dayOfMonths = format(new Date(meeting_date), 'dd');
+  const month = format(new Date(meeting_date), 'MMM');
+  const year = format(new Date(meeting_date), 'yyyy');
```

Comment:
In the `constants.ts` file we have an enum `DateFormats` put the format there with discernable names and use from there.